helm/prometheus-meta-operator/templates/prometheus-rules/coredns.rules.yml

```diff
@@ -67,3 +67,14 @@ spec:
         severity: page
         team: ludacris
         topic: dns
+    - alert: CoreDNSMaxHPAReplicasReached
+      expr: kube_hpa_status_current_replicas{hpa="coredns"} == kube_hpa_spec_max_replicas{hpa="coredns"}
+      for: 120m
+      labels:
+        area: kaas
+        cancel_if_outside_working_hours: "true"
+        severity: page
+        team: se
```

Comment:
cc @giantswarm/ae @giantswarm/sa is this still the correct team for business hours alerts like this?

Reply:
yeah it will be routed to us, not 100% sure who should own it but we can start like this